script/linux-test-vm-bookstore

```diff
@@ -265,7 +266,8 @@ if [[ -n "${BUCKET}" ]] ; then
     -t "${TEST_ID}" \
     -r "${GAE_VERSION}" \
     || error_exit "Could not create ${JSON_FILE}."
-
+  gcloud logging read "resource.labels.project_id=${PROJECT_ID} AND resource.labels.version_id=${GAE_VERSION} AND logName=projects/esp-load-test/logs/appengine.googleapis.com%2Fnginx.error" 2>&1 | tee "${NGINX_ERROR_LOG_FILE}"
```

Comment:
put the code in script/linux-gae-instance so it can be shared.

Reply:
The code has been put in script/linux-gae-instance.